src/GeocodingControl.svelte

```diff
@@ -60,6 +60,10 @@
 
   export let clearOnBlur = false;
 
+  export let clearListOnPick = false;
```

Comment:
Is it intended than now the default behaviour is different?
If the gc should behave as before, it should be `true` - which I think is better to not break other people's apps

Reply:
I'have originally implemented it to work old way as a default, but then changed it because I think it improves UX. I am not sure if I can imagine this to break any existing app.